client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -66,13 +66,28 @@ class CampaignAdminList extends SilverStripeComponent {
     this.props.breadcrumbsActions.setBreadcrumbs(breadcrumbs);
   }
 
+  getSelectedItem() {
+    const itemId = this.props.campaign.changeSetItemId;
+    const items = this.getItems() || [];
+    let selected = null;
+
+    selected = items.find(item => itemId === item.ID);
+
+    if (!selected && items.length > 0) {
+      selected = items[0];
```

Comment:
Based on the current implementation (before this change). If you can load a campaign, the first item in the campaign is selected.